include/openthread.h

```diff
@@ -995,6 +1061,46 @@ int otWriteMessage(otMessage aMessage, uint16_t aOffset, const void *aBuf, uint1
  */
 
 /**
+ * @addtogroup raw-ip6  Raw IPv6
+ *
+ * @brief
+ *   This module includes functions that control raw IPv6 communication.
+ *
+ * @{
+ *
+ */
+
+/**
+ * This function pointer is called when an IPv6 datagram is received.
+ *
+ * @param[in]  aMessage  A pointer to the message buffer containing the received IPv6 datagram.
+ *
+ */
+typedef void (*otRawIp6ReceivedCallback)(otMessage aMessage);
+
+/**
+ * This function registers a callback to provide receivd raw IPv6 datagrams.
+ *
+ * @param[in]  aCallback  A pointer to a function that is called when an IPv6 datagram is received or NULL to disable
+ *                        the callback.
+ *
+ */
+void otSetRawIp6ReceivedCallback(otRawIp6ReceivedCallback aCallback);
+
+/**
+ * This function sends a raw IPv6 datagram via the Thread interface.
+ *
+ * @param[in]  aMessage  A pointer to the message buffer containing the IPv6 datagram.
+ *
+ */
+ThreadError otSendRawIp6(otMessage aMessage);
```

Comment:
I think the word `Raw` here is confusing. How about `otSendIp6Frame()` or `otSendIp6Packet()` (and similar changes for the related functions)?